src/nifgen/metadata/attributes.py

```diff
@@ -228,6 +215,20 @@
 ''',
 },
     },
+    1050322: {
```

Comment:
we need `'codegen-method': 'no'` for this one.

Reply:
Added